doc/api/typescript.md

```diff
@@ -145,6 +161,7 @@ a `node_modules` path.
 [ES Modules]: esm.md
 [Full TypeScript support]: #full-typescript-support
 [`--experimental-strip-types`]: cli.md#--experimental-strip-types
+[`--experimental-type-transform`]: cli.md#--experimental-type-transform
```

Comment:
It occured to me that what about `--experimental-transform-types`, in match with `--experimental-strip-types`?

Reply:
that is correct, 4th time rename 😆